sycl/test-e2e/ESIMD/lit.local.cfg

```diff
@@ -10,5 +10,9 @@ if 'gpu-intel-gen9' in config.available_features and platform.system() == 'Windo
 # The current DG2 postcommit job only runs L0 anyway,
 # so there's no difference in coverage.
 # We should investigate why OCL fails separately.
-if 'gpu-intel-dg2' in config.available_features:
+
+# GPUs to disable ESIMD tests for OpenCL backend.
+gpus_requiring_l0 = ['gpu-intel-dg2', 'gpu-intel-pvc']
+
+if set(gpus_requiring_l0) & set(config.available_features):
```

Comment:
just confirming my own understanding, this works because empty sets are falsey and non-empty sets are truthy right? i don't know python really

Reply:
I think most SYCL developers aren't python experts, so being that smart here is counterproductive for maintainability...